Dockerfile

```diff
@@ -1,36 +1,30 @@
-FROM ubuntu:bionic
+FROM ubuntu:bionic-20191010
 
-RUN apt-get update
```

Comment:
All these separate RUN instructions were merge into a single one. This follows the [good practices of writing a Dockerfile](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/).